docs/.vuepress/config.js

```diff
@@ -78,7 +78,7 @@ module.exports = {
         ]
       },
       {
-        title: 'Annexes',   // required
+        title: 'Appendices',// required
```

Comment:
Looking at where this will end up on the docu site, I think it's most appropriate to use the singular "Appendix" - this is what is used in this context in english, eg extra things at the end of a book or writing. I've never seen the plural "appendices" or "Annexes" at the end of writings :)

Reply:
:+1: then, let's go for "Appendix"